package.json

```diff
@@ -45,7 +45,7 @@
     "typescript": "^3.8.3"
   },
   "lint-staged": {
-    "*.{yml,md}": "prettier --write"
+    "*.{yml,md},!amplify.yml": "prettier --write"
```

Comment:
Running prettier via lint-staged still respects the `.prettierignore` so we don't have to explicitly exclude it